Content/posts/inline-readme-files.md

```diff
@@ -0,0 +1,15 @@
+---
+date: 2020-12-24 12:00
+title: Inline README Files!
+description: Where’s the most important source of information for deciding whether a package is suitable for your needs? It’s in the README file!
+---
+
+If you’ve visited the main package index site over the last week or so, you might have noticed something new. Scroll down a little, and you’ll see the README file for the package, right below all the other information that we aggregate.
+
+When considering adding a new feature, at the heart of that decision has to be the question “Does this feature allow people to make better decisions about the package they’re considering?” Rather embarrassingly, the README file is probably *the most* important piece of information when picking a package dependency. Let’s not focus on that though, and be thankful that we finally got around to adding it! 😅
```

Comment:
I'd say "When considering adding a new feature, at the heart of our decision is the question...", because it's specific to our project, not in general.